https://issues.redhat.com/browse/ACM-8586 (#5853)
open-cluster-management/rhacm-docs: clusters/release_notes/known_issues.adoc

Service unavailable
----

[#manageserviceaccount-addon-limitation]
=== Managed Service Account add-on limitations
//2.9:ACM-8586

The following are known issues and limitations for the `managed-serviceaccount` add-on:

[#manageserviceaccount-does-not-support-ocp]

> ==== The `managed-serviceaccount` does not support {ocp-short} 3.11

Author: In the Adoc Preview I used, I couldn't get this section to render as a subheading like the `installnamespace` and `tolerations and node selector` 

Looking at the text, they're all documented the same way, so I'm not sure why it doesn't render the same way.

//2.9:ACM-8586

> The `managed-serviceaccount` add-on  does not support running on {ocp-short} 3.11. If you run `managed-serviceaccount` add-on on {ocp-short} 3.11, the `managed-serviceaccount` add-on crashes. 

Reviewer: ```suggestion
The `managed-serviceaccount` add-on crashes if you run it on {ocp-short} 3.11. 
```

Reviewer 2: @cadawson i like how you made the statement more concise. I'm not sure if a crash is the actual status. @zhujian7 is it accurate to say that the add-on crashes?

Author: I'll hold off applying the comment until Jian can provide us confirmation. I did see in the Jira it says, "managed-serviceaccount-addon-agent crashed on OCP 311 "

Thanks y'all for the fruitful discussion and looking forward to rounding this issue out!

Reviewer 3: yes, the managed service account addon agent will crash on ocp 311

See link:../../add-ons/configure_nodeselector_tolerations_addons.adoc#configure-nodeselector-tolerations-addons[Configuring nodeSelectors and tolerations for klusterlet add-ons] to learn more about how to use the `addonDeploymentConfig` custom resource to configure `tolerations` and `nodeSelector` for add-ons.

[#manageserviceaccount-addon-limitation]

> === The`managed-serviceaccount` add-on limitation

Reviewer 2: It is best to refer to the component name so that we avoid using monospace in the section title
```suggestion
=== Managed Service Account add-on limitations
```

//2.9:ACM-8586

The following are known issues and limitation for the `managed-serviceaccount` add-on:

- It does not support running on {ocp-short} 3.11. 
- The `installNamespace` field can only have one value. 

> - The `tolerations` and `nodeSelector` settings do not support the `managed-serviceaccount` agent. 

Author: The Jira says, `do not affect" which I interrupted as do not support. Am I correct?

Reviewer 2: Yup, I interrupted like that too

Reviewer 3: The second and third limitations have already been doced in this file [line 446](https://github.com/stolostron/rhacm-docs/pull/5853/files#diff-db9a0499ad3a2863f2248e3b4c575bf63d7a7add5c92a32502fd3b0e8de81072R446-R450) and [line 452](https://github.com/stolostron/rhacm-docs/pull/5853/files#diff-db9a0499ad3a2863f2248e3b4c575bf63d7a7add5c92a32502fd3b0e8de81072R452-R466), could you help organize these 3 limitations?

Author: Please let me know what you think of the newly organized content!
